### ISSUE TITLE
RPM reported as 0 after sleep in Fan Control

### ISSUE THREAD

**Issue by b0ost** [SasaKaranovic/FanControl.OpenFan#3]
After waking up from sleep the fans connected to the OpenFAN Controller are reported as running at 0 RPM (other fans work normally). Unplugging and plugging the USB connector to the controller fixes the issue.

Edit:
This is on a Windows 10 PC with a Gigabyte X570 motherboard

**Comment by SasaKaranovic** [comment]
Hey @b0ost ! Thank you for taking the time to open this issue.
I will take a look into this.

Are you still able to adjust the fan speed?
Or if you restart the FanControl does the issue still persists?

I have tried putting my computer to sleep and after wake-up, controller resumes normal operation.
I noticed when computer goes into sleep, my USB ports are OFF. This might be a BIOS option.

**Comment by b0ost** [comment]
The speeds cannot be adjusted in FanControl when this happens, and a restart of FanControl itself does not fix it either. I noticed that disabling and enabling the USB device also fixes the issue, so at least I don't need to unplug the cable anymore to get this working.

I tried disabling ErP in the BIOS which is supposed to put the USB ports to sleep, but it had no effect. Also the USB selective suspend is disabled in the power plan.
